stream-video-android-ui-core/src/main/kotlin/io/getstream/video/android/ui/common/StreamCallActivity.kt

```diff
@@ -384,8 +384,8 @@ public abstract class StreamCallActivity : ComponentActivity() {
      */
     public open fun onStop(call: Call) {
         logger.d { "Default activity - stopped (call -> $call)" }
-        if (isVideoCall(call) && !isInPictureInPictureMode) {
-            logger.d { "Default activity - stopped: No PiP detected, will leave call. (call -> $call)" }
+        if (isVideoCall(call) && isInPictureInPictureMode) {
```

Comment:
- Lock while in the foreground: does not leave call ✅ 
- Lock while in PiP: leaves call ❓ 

Video [here](https://drive.google.com/file/d/1jUKKryD9w6btLFNuc7WZQdcMdwVBRBky/view?usp=sharing).

Reply:
I think we need to not do anything in this case. I will update the PR.

Reply:
Does not leave the call anymore. ✅